scripts/tests.sh

```diff
@@ -43,7 +43,8 @@ test "${output//[[:blank:]]/}" = "3"
 if [[ "$OSTYPE" == "darwin"* ]]; then
     ETH_PATH="$REPO_ROOT/eth"
 else
-    ETH_PATH="eth"
```

Comment:
I think the option should remain to run it locally or via docker. Maybe checking for `$CI`.